src/js/view/controls/components/chapters.mixin.js

```diff
@@ -14,7 +14,7 @@ class Cue {
         if (this.time.toString().slice(-1) === '%') {
             this.pct = this.time;
         } else {
-            const percentage = (this.time / duration) * 100;
+            const percentage = Math.floor((this.time / duration) * 100);
```

Comment:
This just floors the % to a whole value. It doesn't center the markers based on the precise position they should be placed at (which would be more precise than a whole percentage).

Reply:
I originally thought the same, and will certainly take another look, but in manual testing this was yielding the expected results.

Reply:
1. Multiply then divide (no need for parens and order of operations ensures the highest precision) `100 * this.time / duration`.

2. You want to subtract the percentage of player width that half a cue marker takes up from the percentage already calculated. That will move the marker to the left by half it's width, which is the real fix.

`(100 * this.time / duration) - (50 * markerWidth / playerWidth)`

In your manual testing you should make sure you're not just dealing with positions and durations that seem to work when floored. If they always do please share screenshots that show how the alignment of markers works when floored.